### PR TITLE
Feature/kal list item

### PR DESCRIPTION
**Pull request by MichaelGrosio**

### CODE REVIEW

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-list/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#LIST
+
+##PURPOSE
+
+```html
+<kal-list [datasource]="datasource"
+          [rowTemplate]="rowTemplate"
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-list/README.md:7]: use a directive to access template provided in projected content of kal-list : 
```
<kal-list>
 <ng-template kalListItem let-item="item">
 ....
</ng-template>
</kal-list>
```

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  OnInit,
+  Output,
+  TemplateRef,
+  ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+  selector: 'kal-list',
+  templateUrl: './kal-list.component.html',
+  styleUrls: ['./kal-list.component.sass'],
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts:24]: remove sass file for nested component of main features

File: src/app/app.component.html [hunk 3]
```diff
@@ -15,7 +15,8 @@ <h1>Atoms</h1>
 
   <h1>Molecules</h1>
   <nav>
-    <a routerLink="/molecules/tab-panel">Tab panel</a>
+    <a routerLink="/molecules/tabPanel">Tab panel</a>
```
Comment (ydomenjoud) [inline src/app/app.component.html]: do not use camelCase but kebab-case

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+  /**
+   * Display initials in listing
+   */
+  @Input()
+  get initials(): (item: T) => string {
+    return this.initialsConfig;
+  }
+
+  set initials(value: (item: T) => string) {
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts]: discriminant of items could be other than initials, should rename function name to reflect this possibility

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-tabs/README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
-# TAB PANEL
+#TAB PANEL
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-tabs/README.md:1]: do not modify other files than your feature

**Review by ydomenjoud — CHANGES_REQUESTED** [review]

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-tabs/README.md (hunk 5 above)
Comment (MichaelGrosio) [reply projects/kalidea/kaligraphi/src/lib/molecules/kal-tabs/README.md:1]: Just this readme in PR

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+  /**
+   * Results list
+   */
+  results: T[];
+
+  /**
+   * Datasource to give items list to the component
+   */
+  @Input() datasource: any;
```
Comment (Bellico) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-list/kal-list.component.ts:38]: datasource must implement datasource?

**Review by ydomenjoud — APPROVED** [review]

**Pull request merged**